### PR TITLE
Adding documentation

### PR DESCRIPTION
**Pull request by fmrico**
Hi @lbajo @jginesclavero 

I have added some documentation, as requested @chcorbato in #4 .

Please, review this PR if I should modify anything. You should also make changes to be consistent with this. For example, @lbajo, maybe you should adjust the content of pilot_modes.yaml.

Best 

Signed-off-by: Francisco Martin Rico <fmartin@gsyc.urjc.es>

### CODE REVIEW

**Comment by jginesclavero** [comment]
Hi @fmrico! LGTM for now, I think that this README must contain **how to launch the Pilot** or how to use the .repos file. It's very helpful that you had added a chart. Thank you!

**Comment by jginesclavero** [comment]
# Launching bt_modes_navigator demo.

  Launch gazebo sim and nav2 system (gazebo headless mode ON) -- ```ros2 launch nav2_bringup nav2_tb3_system_modes_sim_launch.py```
  
  Run ```ros2 run system_modes mode-manager [path]/nav2_modes.yaml``` (nav2_modes example in nav2_bt_modes_navigator/system_modes/nav2_modes.yaml) to able the change of nav2 modes.
  
  Finally call the service to change the mode -- ```ros2 service call [/node]/change_mode system_modes/ChangeMode "{node_name: 'node_name', mode_name: 'MODE_NAME'}"``` -- (example ```ros2 service call /bt_navigator/change_mode system_modes/ChangeMode "{node_name: 'navigation2', mode_name: 'BATTERY_CONTINGENCY'}" ```)

**Comment by fmrico** [comment]
Hi @jginesclavero 

Check out how I integrated your instructions. As soon as you review it and it is ok, merge it so anyone could add the remaining instructions (for real robots, for example)

File: README.md [hunk 1]
```diff
@@ -35,4 +35,25 @@ The modes change next parameters in the components (shown in the figure), with s
...
\ No newline at end of file
+![pilot_overview](resources/pilot-urjc.png)
+
+## Launching bt\_modes\_navigator demo
+
+1. Launch gazebo sim and nav2 system (gazebo headless mode ON):
+```
+ros2 launch nav2_bringup nav2_tb3_system_modes_sim_launch.py
+```
+2. Run mode-manager:
```
Comment (jginesclavero) [inline README.md]: This must be: Run mode-manager to able the change of nav2 modes:

File: README.md [hunk 2]
```diff
@@ -35,4 +35,25 @@ The modes change next parameters in the components (shown in the figure), with s
...
+```
+ros2 run system_modes mode-manager [path]/nav2_modes.yaml
+```
+(nav2\_modes example in nav2\_bt\_modes_navigator/system\_modes/nav2\_modes.yaml) to able the change of nav2 modes.
+1. Finally call the service to change the mode:
```
Comment (jginesclavero) [inline README.md]: The third element of the list, 3.

File: README.md [hunk 3]
```diff
@@ -35,4 +35,25 @@ The modes change next parameters in the components (shown in the figure), with s
...
+
+1. Launch gazebo sim and nav2 system (gazebo headless mode ON):
+```
+ros2 launch nav2_bringup nav2_tb3_system_modes_sim_launch.py
+```
+2. Run mode-manager:
+```
+ros2 run system_modes mode-manager [path]/nav2_modes.yaml
+```
+(nav2\_modes example in nav2\_bt\_modes_navigator/system\_modes/nav2\_modes.yaml) to able the change of nav2 modes.
```
Comment (jginesclavero) [inline README.md]: ~~to able the change of nav2 modes~~

**Pull request merged**